plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreConfig.java

```diff
@@ -326,4 +333,30 @@ public ThriftMetastoreConfig setWriteStatisticsThreads(int writeStatisticsThread
         this.writeStatisticsThreads = writeStatisticsThreads;
         return this;
     }
+
+    @Config("hive.metastore.client.http.bearer-token")
+    @ConfigDescription("Http Bearer token to authenticate with a HTTP transport based metastore service")
+    public ThriftMetastoreConfig setHttpToken(String httpToken)
+    {
+        this.httpToken = httpToken;
+        return this;
+    }
+
+    public String getHttpToken()
```

Comment:
Do you envision anything other properties for HTTP based communication?

Maybe another authentication type

Reply:
Right now, I am thinking of supporting a bearer token, but this can be definitely be extended to support more authentication types in the future.